src/style.css

```diff
@@ -1,3 +1,88 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  background-color: #fff;
+}
+
+.nav {
+  display: flex;
+  margin: 0 50px 0 0;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+ul :first-child a {
+  margin: 0 10px 0 0;
+}
+
+ul :last-child a {
```

Comment:
[checklist:](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md#css) Don't use tag names for styling (except html and body)